src/app/components/toast/index.html

```diff
@@ -0,0 +1,92 @@
+      Opens a custom toast and renders the provided component in a toast. The <stache-code>customComponentType</stache-code> parameter is of type <stache-code>Type&lt;SkyToastCustomComponent&gt;</SkyToastCustomComponent></stache-code> and determines the component to be rendered. The optional <stache-code>config</stache-code> parameter is of type <stache-code>SkyToastConfig</stache-code> and adds additional configuration options to the toast. The optional <stache-code>providers</stache-code> parameter is of type <stache-code>Provider[]</stache-code> and specifies an array of custom providers to pass to the <stache-code>customComponentType</stache-code>'s constructor. This method returns the SkyToastMessage instance.
+    </sky-demo-page-property>
+  </sky-demo-page-properties>
+
+  <sky-demo-page-properties sectionHeading="SkyToastConfig type">
+    <sky-demo-page-property
+      propertyName="message"
+      isOptional="true">
+      Sets the message that the toast displays. This may not be provided with the <stache-code>customComponentType</stache-code> property.
+    </sky-demo-page-property>
+    <sky-demo-page-property
+      propertyName="customComponentType"
+      isOptional="true">
+      Sets the component that the toast renders. This may not be provided with the <stache-code>message</stache-code> property.
+    </sky-demo-page-property>
+    <sky-demo-page-property
+      propertyName="providers"
+      isOptional="true">
+      Specifies an array of custom providers to pass to the <stache-code>customComponentType</stache-code>'s constructor.
+    </sky-demo-page-property>
+    <sky-demo-page-property
+      propertyName="toastType"
+      isOptional="true">
+      Sets the type of the toast and determines the color and icon displayed. Available values are: <em>info, success, warning, and danger</em>.
+    </sky-demo-page-property>
+  </sky-demo-page-properties>
+
+  <sky-demo-page-properties sectionHeading="SkyToastMessage type">
+     <sky-demo-page-property
+      propertyName="close()">
+      Closes the linked toast and activates its <stache-code>closing</stache-code> and then <stache-code>closed</stache-code> states.
+    </sky-demo-page-property>
+    <sky-demo-page-property
+      propertyName="isClosing">
+      An observable boolean showing if the toast is currently closing.
+    </sky-demo-page-property>
+    <sky-demo-page-property
+      propertyName="isClosed">
+      An observable boolean showing if the toast is closed.
+    </sky-demo-page-property>
+    <sky-demo-page-property
+      propertyName="message">
+      The message that the toast displays.
+    </sky-demo-page-property>
+    <sky-demo-page-property
+      propertyName="customComponentType">
+      The component that the toast renders.
+    </sky-demo-page-property>
+    <sky-demo-page-property
+      propertyName="providers">
+      An array of custom providers that were passed to the <stache-code>customComponentType</stache-code>'s constructor.
```

Comment:
I thought the `providers` property of `SkyToastConfig` specified this. ... Also, why is this past tense when the `SkyToastConfig` version is present tense?

Reply:
These properties are moved to the ToastMessage entity after the customComponent has been created. So the message is what's currently displayed. The componentType is what is being displayed and the providers are what were passed to it on instantiation.